src/micro_http/src/server.rs

```diff
@@ -507,6 +511,9 @@ impl HttpServer {
     }
 
     /// Adds a stream to the `epoll` notification structure with the `EPOLLIN` event set.
+    ///
+    /// # Errors
+    /// `IOError` is returned when an `epoll::ctl` operation fails.
```

Comment:
Suggestion:
`IOError is returned when an epoll::ctl operation fails.` -> `IOError is returned when the add operation to the epoll instance fails.`

Reply:
@lauralt can I can change like this `IOError is returned when an EPOLL_CTL_ADD operation fails.`. as like we discussed 🙂.